### PR TITLE
Add checkbox component

### PR DESCRIPTION
**Pull request by renancarvalho**
Checkbox for us <3

### CODE REVIEW

File: src/components/ui-checkbox.vue [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
```
Comment (bartderijk) [inline src/components/ui-checkbox.vue:2]: We don't need this div

File: src/components/ui-checkbox.vue [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+import UiLabel from '@/components/ui-label.vue';
+
+export default {
+    components: {
+        UiLabel,
+    },
+    inheritAttrs: false,
+    props: {
+        items: {
+            type: [Array, String],
```
Comment (bartderijk) [inline src/components/ui-checkbox.vue:34]: What scenario do we need a string for?

File: src/stories/ui-checkbox.stories.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            value: 'option3',
+        },
+        {
+            label: 'Option 4',
+            value: 'option4',
+        },
+    ],
+    selected: 'option1',
+    disabled: false,
+    onChange() {},
```
Comment (bartderijk) [inline src/stories/ui-checkbox.stories.js:44]: Do we need the onChange?

**Comment by bartderijk** [comment]
> Checkbox for us <3

Nice work, thanks for lending a hand over here!

File: src/components/ui-checkbox.vue (hunk 1 above)
Comment (renancarvalho) [reply src/components/ui-checkbox.vue:2]: Think we do once the next one is a for.. 

if I remove I get this from eslint - https://eslint.vuejs.org/rules/valid-template-root.html

image

File: src/stories/ui-checkbox.stories.js (hunk 3 above)
Comment (renancarvalho) [reply src/stories/ui-checkbox.stories.js:44]: Was just to validate if it was/is being called. not necessary at all, but there just as an indicator..

File: src/components/ui-checkbox.vue (hunk 1 above)
Comment (bartderijk) [reply src/components/ui-checkbox.vue:2]: I see, thanks for clearifying!
 
Schermafbeelding 2020-09-11 om 13 13 51

**Comment by renancarvalho** [comment]
Can we move it forward?

**Review by bartderijk — APPROVED** [review]

**Pull request merged**